views/peeps/index.erb

```diff
@@ -0,0 +1,261 @@
+<link href="https://fonts.googleapis.com/css?family=Lobster" rel="stylesheet" type="text/css">
```

Comment:
this is BEAUTIFUL and I want to know how to do it please - I couldn't make my style sheet work at all on the RPS challenge :-(
will be checking this out in some detail later on!
I also like how you've laid it out so it is spread out and you can pick out the styling from the HTML, sometimes I've seen some that have looked really cluttered